app/src/main/java/org/sagebionetworks/research/mpower/tracking/recycler_view/SelectionItemViewHolder.java

```diff
@@ -0,0 +1,116 @@
+package org.sagebionetworks.research.mpower.tracking.recycler_view;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.Transformations;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import org.sagebionetworks.research.mpower.R;
+import org.sagebionetworks.research.mpower.tracking.fragment.TrackingFragment;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingItem;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingSection;
+import org.sagebionetworks.research.mpower.tracking.view_model.TrackingTaskViewModel;
+import org.sagebionetworks.research.mpower.tracking.view_model.configs.TrackingItemConfig;
+import org.sagebionetworks.research.mpower.tracking.widget.SelectionUIFormItemWidget;
+
+/**
+ * View Holder for the Selection options from the Selection Screen in a Tracking Task.
+ */
+public class SelectionItemViewHolder extends RecyclerView.ViewHolder {
+    private SelectionUIFormItemWidget widget;
+
+    private TrackingItem trackingItem;
+
+    private LiveData<Boolean> selected;
+
+    private int marginTrackingItem;
+
+    private int marginTrackingSectionTop;
+
+    private TrackingTaskViewModel<? extends TrackingItemConfig, ?> viewModel;
+
+    private TrackingFragment<?, ?, ?> trackingFragment;
+
+    public SelectionItemViewHolder(final SelectionUIFormItemWidget itemView,
```

Comment:
While there is no problem with circular dependencies on java like there is on other languages, usually it is a sign of an architecture issue. You hold the fragment in the view holder, but the fragment also holds the RecyclerView which holds the ViewHolders.  Also, I was under the impression that ViewModels should only be owned within the context of an android lifecycle class i.e. fragment or activity (they get onPause, onResume, etc. called) while a view holder does not get any of those lifecycle calls, and neither does its owner, RecyclerView. 

With RecyclerView/Adapter/ViewHolders, I usually like to do simple listener callbacks for actions that the activity/fragment listen to and then they would call ViewModels.